### ISSUE TITLE
Workshop feedback

### ISSUE THREAD

**Issue by mark-prins** [msupply-foundation/open-msupply#970]
A miscellany of issues observed during client workshops

### Internal order

- [x] Internal order column widths in French
![Screenshot 2022-12-14 at 4 46 51 PM](https://user-images.githubusercontent.com/9192912/208344337-00d1c694-fa15-4a05-9f1b-6cb6d1cf5d26.png)
- [x] Can you autofocus the item input?
- [x] Changing the min MOS so that it is greater than max MOS: perhaps this should update the max too?
- [x] Changing the quantity should refresh the stock evolution chart ( if possible.. it would require saving in order to generate wouldn't it? )
- [x] Can we add a comment? if, for example, some stock is damaged and we need to reduce the quantity
- [x] MOS values should respect regional settings (currently shows `1.5 Months` when the language is French)

### Client requisition

- [x] add AMC please
- [x] the graphs are squashed if there isn't much requested
- [x] Some additional French translating required ('My store', 'Stock on hand', 'other requested')
- [ ] Can we have 'use requested quantity' available per line? as an action?
- [x] If the words are shown on the chart then the vertical alignment of the number isn't great ( would be the same on the internal order probably )
![Screenshot 2022-12-15 at 11 44 31 AM](https://user-images.githubusercontent.com/9192912/208344543-79917ab9-1775-46bb-88fc-0846f99ef917.png)

### Stocktake
- [ ] Make the description text input wider

### Outbound Shipment

- [ ] updating a placeholder row removed all the lines from the modal
- [ ] Can we add a keyboard shortcut for adding an item?

### Reports
- [x] Can we round values?

### Programs

- [ ] Encounter end date input width isn't sufficient to show the helper text
![Screenshot 2022-12-14 at 5 04 06 PM](https://user-images.githubusercontent.com/9192912/208344359-14b5fd3e-46f4-406f-bcf6-38a6d23b905d.png)

### Host
- [x] The order of the stores appears to be case sensitive, not strictly alphabetical (e.g. `Dr`, `General`, `d'Ali`)

### Items
- [ ] Detail page layout doesn't look great now that I look with fresh eyes
![Screenshot 2022-12-15 at 10 32 06 AM](https://user-images.githubusercontent.com/9192912/208344478-2d851f77-e362-4ccf-be6b-0d2cc06b831e.png)

**Comment by DhanyaHerath** [comment]
@mark-prins should we unassign you from this? :) Is it still relevant?

**Comment by mark-prins** [comment]
sill relevant, yes. some of the items are addressed in other issues, but not all.

**Comment by roxy-dao** [comment]
Is this still relevant XD? @mark-prins

**Comment by mark-prins** [comment]
yep - a lot of items on there haven't been addressed. could split this out into multiple issues, probably easier to keep the issue and split to multiple PRs instead though.

**Comment by DhanyaHerath** [comment]
Have these windows changed so much that it is not relevant anymore @mark-prins?

**Comment by mark-prins** [comment]
yep - enough has been done now. only a few minor items remain

**Comment by marthakeezy** [comment]
I think from a QA perspective, changes would have been tested as they were released and the functionality looks so different now, it's probably being tested as a part of regression testing. Moving to done ✅
